cybershell/validation_framework.py

```
@@ -0,0 +1,973 @@
+import hashlib
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)